WalletWasabi/Services/UpdateManager.cs

```diff
@@ -145,6 +195,49 @@ private async Task<(Version Version, string DownloadUrl, string FileName)> GetLa
 		return (githubVersion, url, fileName);
 	}
 
+	private async Task<bool> ValidateWasabiSignatureAsync(string softwareVersion)
+	{
+		var sha256SumsFilePath = Path.Combine(InstallerDir, "SHA256SUMS.asc");
+		var wasabiSigFilePath = Path.Combine(InstallerDir, "SHA256SUMS.wasabisig");
+
+		using HttpClient httpClient = new();
+		string sha256SumsUrl = $"{DownloadURL}/{softwareVersion}/SHA256SUMS.asc";
+		string wasabiSigUrl = $"{DownloadURL}/{softwareVersion}/SHA256SUMS.wasabisig";
+
+		try
+		{
+			var stream = await httpClient.GetStreamAsync(sha256SumsUrl, CancellationToken).ConfigureAwait(false);
```

Comment:
In case of an exception, the stream might be not disposed of. Try to use `using` whenever possible. You can use `using (var stream ...)` with brackets {}.

like you did here https://github.com/zkSNACKs/WalletWasabi/pull/9535/files#diff-3fc9c79accfd2dd9d82aca94e9b518b53671baa674a5f8206415070d56de6e15R152